bundle.py

```diff
@@ -213,20 +212,20 @@ def bundle():
         if WINDOWS:
             patch_wxs()
 
-    # build
-    cmd = ['briefcase', 'build'] + (['--no-docker'] if LINUX else [])
-    subprocess.check_call(cmd)
+        # build
```

Comment:
the briefcase calls need to happen while the pyproject toml is patched, right now it is rolled back too early, resulted in version of the bundle installer set to default 0.0.1

Reply:
that comment would be useful in the code itself, to prevent regressions